process/coordinator/process_test.go

```diff
@@ -2495,20 +2496,20 @@ func TestTransactionCoordinator_VerifyCreatedMiniBlocksShouldErrMaxAccumulatedFe
 	maxGasLimitPerBlock := uint64(1500000000)
 	txHash := []byte("tx_hash1")
 	dataPool := initDataPool(txHash)
-	txCoordinatorArgs:= ArgTransactionCoordinator{
-		Hasher:                            &mock.HasherMock{},
-		Marshalizer:                       &mock.MarshalizerMock{},
-		ShardCoordinator:                  mock.NewMultiShardsCoordinatorMock(3),
-		Accounts:                          initAccountsMock(),
-		MiniBlockPool:                     dataPool.MiniBlocks(),
-		RequestHandler:                    &mock.RequestHandlerStub{},
-		PreProcessors:                     createPreProcessorContainerWithDataPool(dataPool, FeeHandlerMock()),
-		InterProcessors:                   createInterimProcessorContainer(),
-		GasHandler:                        &mock.GasHandlerMock{},
-		FeeHandler:                        &mock.FeeAccumulatorStub{},
-		BlockSizeComputation:              &mock.BlockSizeComputationStub{},
-		BalanceComputation:                &mock.BalanceComputationStub{},
-		EconomicsFee:                      &mock.FeeHandlerStub{
+	txCoordinatorArgs := ArgTransactionCoordinator{
+		Hasher:               &mock.HasherMock{},
+		Marshalizer:          &mock.MarshalizerMock{},
+		ShardCoordinator:     mock.NewMultiShardsCoordinatorMock(3),
+		Accounts:             initAccountsMock(),
+		MiniBlockPool:        dataPool.MiniBlocks(),
+		RequestHandler:       &mock.RequestHandlerStub{},
+		PreProcessors:        createPreProcessorContainerWithDataPool(dataPool, FeeHandlerMock()),
+		InterProcessors:      createInterimProcessorContainer(),
+		GasHandler:           &mock.GasHandlerMock{},
+		FeeHandler:           &mock.FeeAccumulatorStub{},
+		BlockSizeComputation: &mock.BlockSizeComputationStub{},
+		BalanceComputation:   &mock.BalanceComputationStub{},
+		EconomicsFee: &mock.FeeHandlerStub{
```

Comment:
indent

Reply:
This is the way it is indented automatically